### DISCUSSION TITLE
REQUEST: Select range of torrents by holding shift

### DISCUSSION THREAD

**Discussion by SawkeeReemo** [VueTorrent/VueTorrent#1839] [Ideas]
When in torrent selection mode, it would be great if after we select the first one if we could hold shift then select one lower in the stack to be able to select all torrent in between instead of having to click each one individually.

**Comment by Larsluph** [comment]
Isn't this already implemented? Or do you mean something else?

**Reply by SawkeeReemo** [reply]
It is not. I just tried.

**Reply by Larsluph** [reply]
Then we have a new issue to fix related to Safari...

**Reply by SawkeeReemo** [reply]
This is on Chrome, by the way.

**Reply by Larsluph** [reply]
Do you have any kind of errors when looking at the browser console?

**Reply by SawkeeReemo** [reply]
Errors? No, I just can't select a range of torrents while holding shift. It makes you select things one as a time. This specifically in selection mode where you get the check boxes. (not sure what the term is)

**Reply by Larsluph** [reply]
You may need to enable select mode before that

**Comment by Larsluph** [comment]
OK so to recap everything, to use the span selection with the shift key, you first have to enable multi-select mode (either with ctrl-click on a torrent, or by clicking on the checkbox in the toolbar and clicking on a torrent). After that, the shift-click should work and select every torrents between the last selected one and the one you clicked on.

**Reply by SawkeeReemo** [reply]
~~Yeah, that doesn't work. I just tried it on three different systems and browsers. It only selects one at a time.~~

**Reply by SawkeeReemo** [reply]
I'm talking about the checkbox selection. Once you're in that mode, you can't check the box on multiple torrents, only one at a time. That's the feature I'm asking for, to be able to check the box on a range using shift. Just like you can on the standard qBT webui.

**Reply by SawkeeReemo** [reply]
Oh wait! I see what you mean... it just doesn't work when you're trying to click the boxes with shift held down. You have to click the torrents themselves! Sorry, that's a little counter-intuitive. Any chance you can make it so that clicking the boxes with shift held down works?

**Reply by Larsluph** [reply]
Oh yeah I understood now, I should be pretty simple to add, please create an issue and I'll check it out when I have time

**Reply by SawkeeReemo** [reply]
Will do! Thanks for helping me clear that up too.
